### PR TITLE
SORQA-43

### PR DESCRIPTION
**Pull request by pk-sgent**
Fixes #

### CODE REVIEW

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/ContactService.java [hunk 1]
```diff
@@ -126,4 +131,29 @@ public Contact buildEditContact() {
...
+        .wearingMask(YesNoUnknownOptions.NO)
+        .wearingPpe(YesNoUnknownOptions.NO)
+        .otherProtectiveMeasures(YesNoUnknownOptions.NO)
+        .shortDistance(YesNoUnknownOptions.YES)
+        .longFaceToFaceContact(YesNoUnknownOptions.YES)
+        .percutaneous(YesNoUnknownOptions.NO)
+        .contactToBodyFluids(YesNoUnknownOptions.NO)
+        .handlingSamples(YesNoUnknownOptions.NO)
+        .typeOfPlace(TypeOfPlace.HOME)
+        .continent("Africa")
```
[the remark this answers is not in the document]
Comment (pk-sgent) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/ContactService.java:154]: Enum for countries ? ;)

[the remark this answers is not in the document]
Comment (rdutu-vg) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/ContactService.java:154]: Yes, there are several approaches to handle this within an Enum of a continent, having also a list of subcontinents, which based on the same logic can have a list of countries.
Another approach, which might not be the fit for us if we don't cover all countries, is the java Locale library
https://stackoverflow.com/questions/713671/best-way-to-store-country-codes-names-and-continent-in-java

But if it takes too much, we can skip it for now

[the remark this answers is not in the document]
Comment (pk-sgent) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/ContactService.java:154]: There is a big mess in that. In the system, countries for Europe and Western Europe is New Caledonia .. I know that New Caledonia is in French jurisdiction but is still country in Oceania..

[the remark this answers is not in the document]
Comment (rdutu-vg) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/ContactService.java:154]: I see, ok, then let's skip this one for now :D

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**